notes/v1.3.0.md

```diff
@@ -0,0 +1,64 @@
+# Changelog
+
+## Breaking changes
+
+### Removes `pkg/logging` package and sub-packages
+
+Removes the unused `pkg/logging` package and sub-packages since they weren't being used.
+
+## Notable changes
+<!-- Add the list of notable changes, remove otherwise.-->
+
+### New APIs
+<!--  Add the list of new commands, remove otherwise.-->
+
+#### Add `pkg/api/commentapi` package
+
+This new API allows consumers to CRUD comments on various Elastic Cloud platform resources.
+
+#### Add `pkg/api/deploymentapi/extensionapi.Upload` function
+
+### Bug fixes
+<!-- Add the list of bug fixes, remove otherwise.-->
+
+#### Updates the `plan.Track` function to fail more reliably.
+
+Updates the plan tracking functions of the `pkg/plan` package to only fail when
+the plan has actually finished with a failure instead of returning errors found
+in other steps of the plan.
```

Comment:
```suggestion
Updates the plan tracking functions of the `pkg/plan` package to only fail when
the plan has actually finished with a failure, instead of returning errors from
in other steps of the plan.
```